packages/react/src/types/config.ts

```diff
@@ -22,7 +22,9 @@ export type WidgetConfig = {
 	onError?: CallbackFn<IErrorState>
 }
 
-export type Config = IDKitConfig & Required<Pick<IDKitConfig, 'action'>> & WidgetConfig
+export type Config = Required<Pick<IDKitConfig, 'action'>> &
+	WidgetConfig &
+	((Exclude<IDKitConfig, 'app_id'> & { unstable_selfHosted: true }) | (IDKitConfig & { unstable_selfHosted?: false }))
```

Comment:
```suggestion
	((Exclude<IDKitConfig, 'app_id'> & { advanced_selfHosted: true }) | (IDKitConfig & { advanced_selfHosted?: false }))
```

thoughts?

Reply:
feels better to me, would still love to hear @0xPenryn's thoughts on snake_case+camelCase tho

Reply:
aligned sync, we'll do `advanced.self_hosted`